### ISSUE TITLE
Uncaught TypeError: Cannot read property 'tags' of undefined

### ISSUE THREAD

**Issue by prashanthnlingam** [akonwi/git-plus#510]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [git-plus](https://github.com/akonwi/git-plus) package, v5.17.0
### Stack Trace

Uncaught TypeError: Cannot read property 'tags' of undefined

```
At /Users/plingamn/.atom/packages/git-plus/lib/grammars/diff.js:133

TypeError: Cannot read property 'tags' of undefined
    at runCodeGrammar (/Users/plingamn/.atom/packages/git-plus/lib/grammars/diff.js:133:28)
    at Grammar.diffGrammar.tokenizeLine (/Users/plingamn/.atom/packages/git-plus/lib/grammars/diff.js:52:14)
    at TokenizedBuffer.module.exports.TokenizedBuffer.buildTokenizedLineForRowWithText (/Users/plingamn/Downloads/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:494:28)
    at TokenizedBuffer.module.exports.TokenizedBuffer.buildTokenizedLineForRow (/Users/plingamn/Downloads/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:482:19)
    at TokenizedBuffer.module.exports.TokenizedBuffer.tokenizeNextChunk (/Users/plingamn/Downloads/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:287:43)
    at /Users/plingamn/Downloads/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:264:26
    at /Users/plingamn/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/underscore/underscore.js:666:47
```
### Commands

```
     -9:24.5.0 git-plus:difftool (a.inline-block)
     -6:21.1.0 grammar-selector:show (atom-text-editor.editor)
  2x -6:18.8.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -6:03.8.0 git-plus:add (atom-text-editor.editor.is-focused)
  2x -5:47.7.0 git-plus:diff (atom-text-editor.editor.is-focused)
     -5:03.6.0 git-diff:move-to-next-diff (atom-text-editor.editor.is-focused)
     -4:34.4.0 git-diff:toggle-diff-list (atom-text-editor.editor.is-focused)
     -3:56.6.0 git-plus:difftool (atom-text-editor.editor.is-focused)
  2x -3:51.1.0 git-plus:diff (atom-text-editor.editor.is-focused)
     -0:14.7.0 merge-conflicts:detect (atom-text-editor.editor.is-focused)
     -0:00.9.0 git-plus:diff (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
git-blame, v0.4.11 (inactive)
git-plus, v5.17.0 (active)
merge-conflicts, v1.4.4 (active)
minimap, v4.25.0 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (active)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (active)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by code-tree** [comment]
I'm getting this too, probably due to showing syntax highlighting in diff?

**Comment by love8587** [comment]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [git-plus](https://github.com/akonwi/git-plus) package, v5.17.1


### Stack Trace

Uncaught TypeError: Cannot read property 'tags' of undefined

```
At /Users/bhbaek/.atom/packages/git-plus/lib/grammars/diff.js:136

TypeError: Cannot read property 'tags' of undefined
    at runCodeGrammar (/Users/bhbaek/.atom/packages/git-plus/lib/grammars/diff.js:136:26)
    at Grammar.diffGrammar.tokenizeLine (/Users/bhbaek/.atom/packages/git-plus/lib/grammars/diff.js:53:14)
    at TokenizedBuffer.module.exports.TokenizedBuffer.buildTokenizedLineForRowWithText (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:494:28)
    at TokenizedBuffer.module.exports.TokenizedBuffer.buildTokenizedLineForRow (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:482:19)
    at TokenizedBuffer.module.exports.TokenizedBuffer.tokenizeNextChunk (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:287:43)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:264:26
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/underscore/underscore.js:666:47
```

### Commands

```
     -3:31.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -3:25.1.0 core:paste (atom-text-editor#merge-conflicts.gitPath.editor.mini.is-focused)
     -3:23.4.0 core:close (div.package-detail.panels-item)
     -3:19.1.0 pane:reopen-closed-item (atom-text-editor.editor.is-focused)
     -3:18.1.0 core:close (div.package-detail.panels-item)
     -3:17.6.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -3:15.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -3:15.2.0 merge-conflicts:detect (atom-text-editor.editor)
     -2:11.8.0 git-control:toggle (atom-text-editor.editor.is-focused)
     -1:47.7.0 core:close (atom-pane.pane.active)
     -1:44.8.0 git-control:toggle (atom-text-editor.editor.is-focused)
  4x -1:33.3.0 core:close (atom-text-editor.editor.is-focused)
     -1:08.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:06.8.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:04.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:04.1.0 git-plus:log (atom-text-editor.editor)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.29.12 (inactive)
autocomplete-modules, v1.6.3 (active)
git-control, v0.8.2 (active)
git-log, v0.4.1 (active)
git-plus, v5.17.1 (active)
git-time-machine, v1.5.3 (inactive)
linter, v1.11.16 (active)
linter-jshint, v2.1.0 (active)
merge-conflicts, v1.4.4 (active)
node-debugger, v1.7.0 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (active)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (active)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (active)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by akonwi** [comment]
This has been fixed in version 5.20.0 published today
